README.rst

```diff
@@ -45,76 +39,12 @@ Distribution: https://www.anaconda.com/download
 
    $ conda install numba
 
-If you wanted to compile Numba from source,
-it is recommended to use conda environment to maintain multiple isolated
-development environments.  To create a new environment for Numba development::
-
-   $ conda create -p ~/dev/mynumba python numpy llvmlite
-
-To select the installed version, append "=VERSION" to the package name,
-where, "VERSION" is the version number.  For example::
-
-   $ conda create -p ~/dev/mynumba python=2.7 numpy=1.11 llvmlite
-
-to use Python 2.7 and Numpy 1.11.
-
-If you need CUDA support, you should also install the CUDA toolkit::
-
-   $ conda install cudatoolkit
-
-This installs the CUDA Toolkit version 8.0, which requires driver version 375.x
-or later to be installed.
-
-Custom Python Environments
---------------------------
-
-If you're not using conda, you will need to build llvmlite yourself:
-
-Building and installing llvmlite
-''''''''''''''''''''''''''''''''
-
-See https://github.com/numba/llvmlite for the most up-to-date instructions.
-You will need a build of LLVM 6.0.x.
-
-::
-
-   $ git clone https://github.com/numba/llvmlite
-   $ cd llvmlite
-   $ python setup.py install
-
-Installing Numba
-''''''''''''''''
-
-::
-
-   $ git clone https://github.com/numba/numba.git
-   $ cd numba
-   $ pip install -r requirements.txt
-   $ python setup.py build_ext --inplace
-   $ python setup.py install
-
-or simply
-
-::
-
-   $ pip install numba
-
-If you want to enable CUDA support, you will need to install CUDA Toolkit 8.0.
-After installing the toolkit, you might have to specify environment variables
-in order to override the standard search paths:
-
-NUMBAPRO_CUDA_DRIVER
-  Path to the CUDA driver shared library
-NUMBAPRO_NVVM
-  Path to the CUDA libNVVM shared library file
-NUMBAPRO_LIBDEVICE
-  Path to the CUDA libNVVM libdevice directory which contains .bc files
-
+For more options, see the Installation Guide: http://numba.pydata.org/numba-doc/latest/user/installing.html
```

Comment:
Couple of lines above this there's a `numba` in `The easiest way to install numba and ` that should have a capital `N`